packages/open-next/src/build/createServerBundle.ts

```diff
@@ -153,14 +158,26 @@ async function generateBundle(
   buildHelper.copyEnvFile(appBuildOutputPath, packagePath, outputPath);
 
   // Copy all necessary traced files
-  await copyTracedFiles(
+  const { tracedFiles, manifests } = await copyTracedFiles(
     appBuildOutputPath,
     packagePath,
     outputPath,
     fnOptions.routes ?? ["app/page.tsx"],
     isBundled,
   );
 
+  await applyCodePatches(options, Array.from(tracedFiles), manifests, [
+    // TODO: create real code patchers here
+    {
+      name: "fakePatchChunks",
+      filter: /chunks\/\d*\.js/,
```

Comment:
```suggestion
      filter: /chunks\/\d+\.js/,
```
Also use platform independent regex